lib/src/executables/react16_dependency_override_update.dart

```diff
@@ -27,14 +32,23 @@ Then, review the the changes and commit.
 """;
 
 void main(List<String> args) {
+  final reactVersionConstraint =
+      VersionConstraint.parse(reactVersionRangeForTesting);
+  final overReactVersionConstraint =
+      VersionConstraint.parse(overReactVersionRangeForTesting);
+
   final pubspecYamlQuery = FileQuery.dir(
     pathFilter: (path) => p.basename(path) == 'pubspec.yaml',
     recursive: true,
   );
 
   exitCode = runInteractiveCodemod(
     pubspecYamlQuery,
-    Ignoreable(DependencyOverrideUpdater()),
```

Comment:
Is `DependencyOverrideUpdater` not used anymore? Can we remove it?

Reply:
same question...

Reply:
We could remove it - but if we ever needed to do an update that involved dependency_overrides - we'd have to rewrite it.